desktop/src/styles.scss

```
@@ -0,0 +1,2176 @@
+  color: var(--text-dim);
+  letter-spacing: 0.08em;
+  font-weight: 650;
+}
+
+/* ── Code/Log panels ───────────────────────────────────────────────────── */
+
+.ascii-panel,
+.json-panel {
+  margin: 0;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)